packages/root-cms/core/app.tsx

```diff
@@ -3,6 +3,7 @@ import path from 'node:path';
 import {fileURLToPath} from 'node:url';
 import {Request, Response, RootConfig} from '@blinkk/root';
 import {render as renderToString} from 'preact-render-to-string';
+import packageJson from '../package.json' assert {type: 'json'};
```

Comment:
i'm a little uncertain about how this import works when compiled, will need to test/verify on a non-crucial project (e.g. on the rootjs.dev site).

Reply:
I ran the project locally and used it against the docs package and it seemed to work. I think the local build (via `pnpm run build` at the project root) bundles things the same way as in prod?